main/guides/orchestration/getting-started/key-concepts.md

```diff
@@ -1,87 +1,67 @@
-
-It simplifies accessing and interacting with multiple chains, providing a unified interface for the orchestration logic to manage cross-chain operations effectively.
-ChainHub also allows dynamic registration and use of chain and connection information.
+- `asAmount` converts a denom amount to an `Amount` with a brand. See [asAmount](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.Orchestrator#asAmount).
 
 ```javascript
-const chainHub = makeChainHub(remotePowers.agoricNames)
+const amount = orchestrator.asAmount({ denom: 'uatom', value: 1000n })
+```
 
-// Register a new chain with its information
-chainHub.registerChain(chainKey, chainInfo)
+## Orchestration Account
 
-// Register a connection between the Agoric chain and the new chain
-chainHub.registerConnection(
-  agoricChainInfo.chainId,
-  chainInfo.chainId,
-  connectionInfo
-)
-```
+Orchestration accounts are a key concept in the Agoric Orchestration API, represented by the [`OrchestrationAccountI`](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.OrchestrationAccountI)
+interface. These accounts provide high-level operations for managing accounts on remote chains, allowing seamless
+interaction and management of interchain accounts. The orchestration accounts abstract the complexity of interchain
+interactions, providing a unified and simplified interface for developers.
 
-In this example, `chainHub` is used to register a new chain and establish a connection between the Agoric chain and the newly registered chain.
+### Account Creation
 
-### Orchestration Account
+- `makeAccount` (for a chain object) creates a new account on local and/or remote chain as below.
 
-Orchestration accounts are a key concept in the Agoric Orchestration API, represented by the [`OrchestrationAccountI`](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.OrchestrationAccountI) interface. These accounts provide high-level operations for managing accounts on remote chains, allowing seamless interaction and management of interchain accounts. The orchestration accounts abstract the complexity of interchain interactions, providing a unified and simplified interface for developers.
+```javascript
+const [agoric, remoteChain] = await Promise.all([
+  orch.getChain('agoric'),
+  orch.getChain(chainName)
+])
+const [localAccount, remoteAccount] = await Promise.all([
+  agoric.makeAccount(),
+  remoteChain.makeAccount()
+])
+```
 
-**1. Address Management**
+### Address Management
 
 - `getAddress` retrieves the address of the account on the remote chain.
 
 ```javascript
 const address = await orchestrationAccount.getAddress()
 ```
 
-**2. Balance Management**
+### Balance Management
```

Comment:
What is happening under the hood here, is IF we invoke `getBalance` on an remote account, this is actually an Interchain Query (ICQ). We can probably take the opportunity to briefly explain this here if this is the first time they see it. 

Example from orch dapp contract:
```js
const remoteChainBalance = await remoteAccount.getBalance('uosmo');
```